lib/vue-simple-suggest.vue

```diff
@@ -515,12 +543,21 @@ export default {
     },
     clearSuggestions () {
       this.suggestions.splice(0)
+    },
+    getId (value, i) {
+      return `${this.listId}-suggestion-${this.isPlainSuggestion ? i : this.valueProperty(value)}`
     }
   }
 }
 </script>
 
 <style>
+.vue-simple-suggest .suggestions {
```

Comment:
I took the approach to remove default ul styling even if `destyled` is used.

Reply:
What are reasons behind this decision?

Reply:
Is it because it's not generally intended for suggestions to have a list-like appearance?

Reply:
> Is it because it's not generally intended for suggestions to have a list-like appearance?

That's it. But using a selector `.vue-simple-suggest ul` like you suggested before actually makes more sense because it's specificity score would be lower than `.vue-simple-suggest .suggestions`. I just made this change.

Reply:
Awesome! Great job right there! :smiley: